src/schema/slack/functions/_scripts/src/templates/template_mod.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  autogeneratedComment,
+  getFunctionName,
+  renderFunctionImport,
+} from "./template_utils.ts";
+import { FunctionRecord } from "../types.ts";
+
+const renderFunctionPropWithType = (defineFunctionInput: FunctionRecord) =>
+  `${getFunctionName(defineFunctionInput.callback_id)}: ${
+    getFunctionName(defineFunctionInput.callback_id)
+  }`;
```

Comment:
nit: I think we could get away with just a single call here using [Object initializer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)
 
```suggestion
const renderFunctionPropWithType = (defineFunctionInput: FunctionRecord) =>
  `${getFunctionName(defineFunctionInput.callback_id)}`;
```

Reply:
Noice, I felt like this function was ugly I'm happy to remove it